website/docs/r/spot_instance_request.html.markdown

```diff
@@ -61,6 +61,11 @@ Spot Instance Requests support all the same arguments as
   The duration period starts as soon as your Spot instance receives its instance ID. At the end of the duration period, Amazon EC2 marks the Spot instance for termination and provides a Spot instance termination notice, which gives the instance a two-minute warning before it terminates.
   Note that you can't specify an Availability Zone group or a launch group if you specify a duration.
 * `instance_interruption_behavior` - (Optional) Indicates whether a Spot instance stops or terminates when it is interrupted. Default is `terminate` as this is the current AWS behaviour.
+* `valid_until` - The end date and time of the request, in UTC ISO8601 format
```

Comment:
Once we switch to RFC3339, we should link to its documentation, e.g. `in [RFC3339 format](https://tools.ietf.org/html/rfc3339#section-5.8)`

Reply:
Updated the document with RFC3339 change.